base.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     es6: true,
     node: true,
   },
-  parser: 'babel-eslint', // allows both flowtype and static class properties
+  parser: '@typescript-eslint/parser',
```

Comment:
does the typescript parser work with non-typescript files?

Reply:
Yes, it works on plain JavaScript files too.